src/components/CommentsByArticleId.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react'
+import axios from 'axios';
+import CommentCard from './CommentCard';
+export default function CommentsByArticleId({id, activeUserName}){
+    let articleId = id;
+    const [commentId, setCommentId] = useState(null)
+    const [commentsVote, setCommentsVote] = useState(null)
+    const [commentsList, setCommentsList] = useState([])
+
+    useEffect(()=>{
+        axios.get(`https://nc-news-24h6.onrender.com/api/articles/${id}/comments`).then((response)=>{
+            setCommentsList(response.data.comments)
+        })
+    }, [])
+
+    return(
+        <div className="commentsBlock">
```

Comment:
You could add a loading pattern so the user is given an indication when the comments are loading, here are some notes on using an `isLoading` state to conditionally render a loading message https://notes.northcoders.com/courses/js-front-end/useEffect#loading-patterns